tools/build-image.sh

```diff
@@ -27,7 +27,10 @@ usage: $0 [options]
 options:
   -n NAME | --name NAME     Specify build name (default: aquarium).
   -c | --clean              Cleanup an existing build directory before building.
-  -t | --type IMGTYPE       Specify image type (default: vagrant)
+  -t | --type IMGTYPE       Specify image type (default: vagrant).
```

Comment:
add a `container` type  ?? :grin:

Reply:
we're not actually building a container with aquarium in it, so it feels weird adding this to the build script (which is actually used to build aquarium by the container's entrypoint). IMO, this might need a script that calls either on the container builder or on `build-image.sh` though.